### ISSUE TITLE
Game window text highlight color should be a user setting, or at least changed

### ISSUE THREAD

**Issue by friscodelrosario** [billforsternz/tarrasch-chess-gui#27]
I don't care that I can't change text color in the moves window. (I use Sublime Text, for which a bajillion color schemes are available, so I get more than enough choices in changing text color). I even like the default blue. But the highlighted text color has to be made darker.

The text highlight is so light that the selected, reversed text can't be seen.  If I can do that at the system level, I can't find it in Ubuntu or Elementary OS forums (where the users are all concerned with changing system colors).

Sure, the highlighted text is only unreadable until any user action undoes the highlight, but while editing text, a writer will sit and stare at highlighted text for minutes, because that's the text he's thinking about changing.

![Screenshot from 2020-09-04 02 20 53](https://user-images.githubusercontent.com/59985/92223948-0c595280-ee56-11ea-8a79-56d869888978.png)

**Comment by billforsternz** [comment]
It's a shame that it looks that way in your Linux environment. On Windows it is white on blue (literally the inverse of normal text) and very readable. I'll leave the issue open, but I warn you that I don't really have time or motivation to track down issues in obscure Linux distributions. That's one of the downsides of being a Linux user. I will take a look in Ubuntu next time I fire up my Ubuntu system.

**Comment by billforsternz** [comment]
Fired up my Ubuntu system to look at issue #20 and noted that the selected text is white on orange there just as in all other Ubuntu applications. Implication is that this is an Operating system setting.

**Comment by friscodelrosario** [comment]
You're right. Elementary OS highlights text acceptably in ChessX, but not Tarrasch. I'll mention it in an Elementary forum.

![Screenshot from 2020-09-15 03 56 13](https://user-images.githubusercontent.com/59985/93202477-997a9200-f707-11ea-8762-a9e81c78b997.png)
